grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
```

Comment:
Looks like we can use this, if we don't want to inline our config and add another global config file:

```js
    options: {
      htmllintrc: '.htmllintrc'
    },
```